doc/commands.md

```diff
@@ -193,9 +194,13 @@ See the [Implementation notes](implementation_notes.md#building-latex-files) for
 
 **Flags**
 
-- `--order`: The order of the problems, e.g. `BDCA`. Can be used to order problems from easy to difficult. When labels have multiple letters, `B1,A1,A2,B2` is also allowed.
 - `--cp`: Instead of symlinking the final pdf, copy it into the contest directory.
+- `--order`: The order of the problems, e.g. `BDCA`. Can be used to order problems from easy to difficult. When labels have multiple letters, `B1,A1,A2,B2` is also allowed.
+- `--order-from-ccs`: Order the problems by increasing difficulty, extracted from the api, e.g.: https://www.domjudge.org/demoweb. Defaults to value of ccs_url in contest.yaml.
```

Comment:
:+1:

Reply:
I noticed these were missing, so I just copied the descriptions from the `--help` command :joy: